features/step_definitions/social_media_steps.rb

```diff
@@ -37,7 +37,7 @@
     social_container = Organisation.last
     visit organisation_path(social_container)
   end
-  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: "Connect with #{social_container.display_name} on #{social_service}")
+  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: "#{social_service}")
```

Comment:
Picky, and just realised this:

```suggestion
  assert page.has_css?(".social-media-accounts .social-media-link.#{social_service.parameterize}", text: social_service)
```